app/src/shared/components/TimeSeriesGraph/timeSeriesGraph.stories.jsx

```diff
@@ -18,10 +18,10 @@ const stories =
         }))
         .addDecorator(withKnobs)
 
-const today = new Date().getTime()
-
 const MSEC_DAILY = 86400000
 
+const today = new Date('2035-01-01').getTime() + Math.floor(MSEC_DAILY / 2)
```

Comment:
Graphs have to pick those well. Noon of each day is perfect cause it aligns nicely.